js/src/forum/index.js

```diff
@@ -37,12 +26,12 @@ app.initializers.add('justoverclock/flarum-ext-welcomebox', () => {
             m('div', { class: 'iconbadge' }, listItems(user.badges().toArray())),
             m('.ulwb', { class: 'contentwb' }, [
               m('li', [
-                m('label', { class: 'textinfo' }, app.translator.trans('flarum-ext-welcomebox.forum.npost')),
+                m('label', { class: 'textinfo' }, app.translator.trans('core.forum.user.posts_link')),
```

Comment:
This change is wrong. You should not use core translations directly. You should use your own key (as before), but reference core translation in translation file (as `=> core.forum.user.posts_link` - in this way translators have more flexibility and can translate this phrase differently than in generic phrase used by core (context matters). This is explained in docs: https://docs.flarum.org/extend/i18n.html#appendix-b-coding-for-the-world

Reply:
Thanks @rob006,I'll get that fixed up in another PR; that pretty much clears up any questions/concerns about translations (and what was said in the docs).